sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -250,9 +250,14 @@ private[hive] object SparkSQLCLIDriver extends Logging {
 
     var ret = 0
     var prefix = ""
-    val currentDB = ReflectionUtils.invokeStatic(classOf[CliDriver], "getFormattedDb",
-      classOf[HiveConf] -> conf, classOf[CliSessionState] -> sessionState)
-
+    def currentDB = {
```

Comment:
Could we change it to?
```scala
    val currentDB = if (HiveConf.getBoolVar(conf, HiveConf.ConfVars.CLIPRINTCURRENTDB)) {
      val currDb = SparkSQLEnv.sqlContext.sessionState.catalog.getCurrentDatabase
      s" ($currDb)"
    } else {
      ""
    }
```

Reply:
I will keep it as def instead of val , if user change the database then it should take the updated DB. Indentation I changed as per your suggestion